zerver/migrations/0201_realm_message_content_allowed_in_email_notifications.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe it adds convenience or you should always check the latest migration number merged in master and renumber your PR accordingly.
we have a tool for this purpose: `tools/renumber-migrations`

Reply:
Yep, sure!